modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -208,7 +208,7 @@ public function finalStep(&$state)
 
         \SimpleSAML\Auth\Source::completeAuth($state);
 
-        return $attributes;
+        return [$username, $attributes];
```

Comment:
OK, you're probably not gonna like me, but I've taken a closer look here and I think the return-statement should go away completely, along with the phpdoc..
This function is called only once [here](https://github.com/simplesamlphp/simplesamlphp/blob/9a988ee0e2f7bc7d997a47188d87a43e5928e61e/modules/cas/www/linkback.php#L26) and a return-value is never used..

Reply:
No problem, thanks for the info. I've updated the phpDoc/code.